config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_log_event_attributes.conf

```diff
@@ -0,0 +1,53 @@
+version = 1
+name = log_event_attributes
+commands = [
+  {
+    upgrade: [
+      {
+        type: Attribute,
+        action: ADD,
+        data: {
+          attributes = [
+            {
+              fqn: LOG_EVENT.timestampNanos,
+              key: timestampNanos,
+              value_kind: TYPE_TIMESTAMP,
+              unit: ns,
+              display_name: Timestamp,
+              scope_string: LOG_EVENT,
+              sources: [QS],
+              type: ATTRIBUTE
+            },
+            {
+              fqn: LOG_EVENT.spanId,
+              key: spanId,
+              value_kind: TYPE_STRING,
+              display_name: Span ID,
+              scope_string: LOG_EVENT,
+              sources: [QS],
+              type: ATTRIBUTE
+            },
+            {
+              fqn: LOG_EVENT.traceId,
+              key: traceId,
+              value_kind: TYPE_STRING,
+              display_name: Trace ID,
+              scope_string: LOG_EVENT,
+              sources: [QS],
+              type: ATTRIBUTE
+            },
+            {
+              fqn: LOG_EVENT.attributes,
+              key: attributes,
+              value_kind: TYPE_STRING_MAP,
+              display_name: Attributes,
+              scope_string: LOG_EVENT,
```

Comment:
shouldn't this be just a `scope`? Did we change it to `scope_string`?

In the existing, I still see the name `scope` - https://github.com/hypertrace/config-bootstrapper/blob/bdd3c82f093204e43d8a92f8d7615479c07c4bfa/config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace-core/1_trace_attributes.conf#L17

Reply:
Yes, that is deprecated.

Reply:
I saw the above comments - https://github.com/hypertrace/config-bootstrapper/pull/51#discussion_r614064042